source/_integrations/webostv.markdown

```diff
@@ -49,14 +49,17 @@ Automations can also be created using an automation action:
 wake_on_lan: # enables `wake_on_lan` integration
 
 automation:
-  trigger:
-    - platform: webostv.turn_on
-      entity_id: media_player.lg_webos_smart_tv
-  action:
-    - service: wake_on_lan.send_magic_packet
-      data:
-        broadcast_address: 192.168.1.255
-        mac: AA-BB-CC-DD-EE-FF
+  - alias: Turn On Living Room TV with WakeOnLan
+    trigger:
+      - platform: device
```

Comment:
Added @frenck for an additional opinion since I have less experience in this

Reply:
I'm not understanding why we should use device automation here?
in general, device automation are less friendly in YAML use case.

Could you explain what was wrong with the existing automation @igrybkov?